sdk/monitor/Azure.ResourceManager.Monitor/api/Azure.ResourceManager.Monitor.netstandard2.0.cs

```diff
@@ -780,6 +786,52 @@ public partial class MonitorPrivateLinkScopeResource : Azure.ResourceManager.Arm
+        public virtual Azure.Response<bool> Exists(string azureMonitorWorkspaceName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<bool>> ExistsAsync(string azureMonitorWorkspaceName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.ResourceManager.Monitor.MonitorWorkspaceResource> Get(string azureMonitorWorkspaceName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Pageable<Azure.ResourceManager.Monitor.MonitorWorkspaceResource> GetAll(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.AsyncPageable<Azure.ResourceManager.Monitor.MonitorWorkspaceResource> GetAllAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.Monitor.MonitorWorkspaceResource>> GetAsync(string azureMonitorWorkspaceName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        System.Collections.Generic.IAsyncEnumerator<Azure.ResourceManager.Monitor.MonitorWorkspaceResource> System.Collections.Generic.IAsyncEnumerable<Azure.ResourceManager.Monitor.MonitorWorkspaceResource>.GetAsyncEnumerator(System.Threading.CancellationToken cancellationToken) { throw null; }
+        System.Collections.Generic.IEnumerator<Azure.ResourceManager.Monitor.MonitorWorkspaceResource> System.Collections.Generic.IEnumerable<Azure.ResourceManager.Monitor.MonitorWorkspaceResource>.GetEnumerator() { throw null; }
+        System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
+    }
+    public partial class MonitorWorkspaceResourceData : Azure.ResourceManager.Models.TrackedResourceData
+    {
+        public MonitorWorkspaceResourceData(Azure.Core.AzureLocation location) : base (default(Azure.Core.AzureLocation)) { }
+        public string AccountId { get { throw null; } }
+        public Azure.ResourceManager.Monitor.Models.MonitorWorkspaceDefaultIngestionSettings DefaultIngestionSettings { get { throw null; } }
+        public Azure.ETag? ETag { get { throw null; } }
+        public Azure.ResourceManager.Monitor.Models.MonitorWorkspaceMetrics Metrics { get { throw null; } }
+        public System.Collections.Generic.IReadOnlyList<Azure.ResourceManager.Monitor.Models.PrivateEndpointConnectionAutoGenerated> PrivateEndpointConnections { get { throw null; } }
```

Comment:
We have an `AutoGenerated` and it is unacceptable.
We should remove the deduplication flag and resolve the duplicate schema using directives.

Reply:
I have tried running dotnet build with the following added under `directive:`
```
  - from: monitoringAccounts_API.json
    where: $.definitions.AzureMonitorWorkspace.properties.privateEndpointConnections
    transform: $["x-ms-client-name"] = "MonitorWorkspacePrivateEndpointConnections"
```
and similar variations, but I am not noticing any change in the duplication errors. Can you suggest a fix?

Reply:
I have resolved it by adding directives for v4 assets

```
  - from: v4/privatelinks.json
    where: $.definitions.PrivateEndpoint
    transform: $["x-ms-client-name"] = "CommonPrivateEndpointV4"
  - from: v4/privatelinks.json
    where: $.definitions.PrivateEndpointConnection
    transform: $["x-ms-client-name"] = "CommonPrivateEndpointConnectionV4"
  - from: v4/privatelinks.json
    where: $.definitions.PrivateEndpointConnectionProperties
    transform: $["x-ms-client-name"] = "CommonPrivateEndpointConnectionPropertiesV4"
  - from: v4/types.json
    where: $.definitions.Resource
    transform: $["x-ms-client-name"] = "CommonResourceV4"
```